### DISCUSSION TITLE
Yaşam, Evren ve Her Şey

### DISCUSSION THREAD

**Discussion by Ersan0102** [sonsuzus/sonsuzus#427] [Soru Cevap]
**Problem**
Programınız Yaşamın, Evrenin ve Her Şeyin Cevabını bulmak için kaba kuvvet yaklaşımını kullanmaktır. Daha doğrusu... küçük sayıları girişten çıkışa yeniden yazın. 42 sayısını okuduktan sonra giriş işlemini durdurun. Girişteki tüm sayılar bir veya iki basamaklı tam sayılardır.

**Örnek Giriş:**
1
2
88
42
99

**Örnek Çıktı:**
1
2
88

**Comment by Ersan0102** [comment]
Linki: https://www.hackerearth.com/practice/basic-programming/input-output/basics-of-input-output/practice-problems/algorithm/life-the-universe-and-everything/

**Comment by Erva-Naz** [comment]
```c++
#include <iostream>
using namespace std;
int main() {
	int i=0;

	while(i!=42){
		cin>>i;
		if(i!=42) {
			cout<<i<<endl;
		}

}	
}
```
